internal/impl/devbox.go

```diff
@@ -299,11 +305,6 @@ func (d *Devbox) NixEnv(ctx context.Context, includeHooks bool) (string, error)
 func (d *Devbox) EnvVars(ctx context.Context) ([]string, error) {
 	ctx, task := trace.NewTask(ctx, "devboxEnvVars")
 	defer task.End()
-	// this only returns env variables for the shell environment excluding hooks
```

Comment:
hmm I think this comment was actually for `nixEnv`, right? If so, could we leave it in place?

Reply:
Its explaining why Mohsen did `nixEnv` and not `NixEnv`.